src/commands/docker.ts

```diff
@@ -112,7 +111,9 @@ export async function runWorkflowWithProgress(
           });
           // early returns handled within each workflow depending on how far it got
         });
-
+        progress.report({
+          message: `${start ? "Starting" : "Stopping"} ${workflow.resourceKind}...`,
+        });
```

Comment:
New placeholder message so we shouldn't see a plain `Local:` progress notification during slower init